torch/csrc/jit/codegen/cuda/lower_predicate.cpp

```diff
@@ -142,19 +142,24 @@ class ConditionalFromPredicateModifier {
     switch (pred->predicate_type()) {
       case PredicateType::Inline: {
         return PredicateCompute::getInlinePredicate(
-            pred->expr(), for_loops_structure_, pred->thread_pred());
+            pred->expr(),
+            for_loops_structure_,
+            pred->thread_pred(),
+            PredicateType::Inline);
       }
       case PredicateType::Misaligned: {
         return PredicateCompute::getInlinePredicate(
             pred->expr(),
             for_loops_structure_,
             pred->thread_pred(),
-            true,
-            true);
+            PredicateType::Misaligned);
       }
       case PredicateType::InternalSync: {
         return PredicateCompute::getInlinePredicate(
-            pred->expr(), for_loops_structure_, pred->thread_pred(), false);
+            pred->expr(),
+            for_loops_structure_,
+            pred->thread_pred(),
+            PredicateType::InternalSync);
```

Comment:
These can be combined like:

```suggestion
   case PredicateType::Inline: 
   case PredicateType::Misaligned:
   case PredicateType::InternalSync: {
        return PredicateCompute::getInlinePredicate(
            pred->expr(),
            for_loops_structure_,
            pred->thread_pred(),
            pred->predicate_type());
    }
```